charts/hdfs-datanode-k8s/README.md

```diff
@@ -16,7 +16,7 @@ HDFS `datanodes` running inside a kubernetes cluster. See the other chart for
      It will be supplied below as the `clusterDnsIP` parameter.
 
   ```
-  $ kubectl get svc --all-namespaces | grep kube-dns
```

Comment:
Ah. I think we should revert to the previous line since kube-dns is not in the `default` namespace.